c4p2n/models.py

```diff
@@ -1,10 +1,19 @@
 from abc import abstractmethod
 from datetime import datetime
+from functools import wraps
 from typing import Dict, Any, List, Union
 
 from pydantic import BaseModel, Field, validator
 
 
+@wraps(Field)
+def EmptyStringField(**kwargs: Any) -> Any:
```

Comment:
`-> Field` would be better, I think

Reply:
It doesn't return `Field`, `Field` is a pydantic function. `EmptyStringField` here wraps it and returns a result of it's call with overridden `default`.

Reply:
Ok, I see, thanks!